bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1428,15 +1429,15 @@ namespace JsRTApiTest
         REQUIRE(JsSetPromiseContinuationCallback(PromiseContinuationCallback, &callback) == JsNoError);
         REQUIRE(JsRunScript(
             _u("new Promise(") \
-            _u("  function(resolve, reject) {") \
-            _u("      resolve('basic:success');") \
-            _u("  }") \
+            _u("    function(resolve, reject) {") \
```

Comment:
Was this intentional?  It seems like JS tends to use two spaces for indentation anyway, so I'd rather just leave this as it was unless there's a strong opinion to the contrary.